docs/reference/contrib/postgres_search.rst

```diff
@@ -13,23 +13,20 @@ This contrib module provides a search engine backend for Wagtail using
 
 **Features**:
 
-- Supports all the search features available in Wagtail.
+- It supports all the search features available in Wagtail.
 - Easy to install and adds no external dependency or service.
-- Excellent performance for sites with up to 200 000 pages.
-  Stays decent for sites up to a million pages.
-- Faster to reindex than Elasticsearch if you use PostgreSQL 9.5 or more.
+- Excellent performance for sites with up to 200 000 pages and stays decent for sites up to a million pages.
+- Faster to reindex than Elasticsearch, if you use PostgreSQL 9.5 or higher.
 
-The only known **downsides** concern :
+**Drawbacks**:
 
-**Downsides**:
-
-- ``SearchField(partial_match=True)`` is not handled.
+- Partial matching (``SearchField(partial_match=True)``) is not supported
 - Due to a PostgreSQL limitation, ``SearchField(boost=…)`` is only partially
-  respected. It is changed so that there can only be 4 different boosts.
-  If you define 4 or less different boosts,
+  respected. It is changed so that there can only be four different boosts.
+  If you define four or less different boosts,
```

Comment:
If you define four or less different boosts -> If you define up to four different boosts